### ISSUE TITLE
hcitool and hcidump have been deprecated upstream

### ISSUE THREAD

**Issue by thomwiggers** [nirmankarta/PyBeacon#61]
From the [Arch wiki](https://wiki.archlinux.org/index.php/bluetooth#Installation):

>Some tools such as hcitool and hciconfig have been deprecated upstream, and are no longer included in bluez-utils. Since these tools will no longer be updated, it is recommended that scripts be updated to avoid using them. If you still desire to use them, install bluez-utils-compatAUR. See FS#53110 and the [Bluez mailing list](https://www.spinics.net/lists/linux-bluetooth/msg69239.html) for more information.

**Comment by PrabhanshuAttri** [comment]
@thomwiggers Yes, I remember this. I remember that bluez-hcidump package used to be a part of bluez package. While developing PyBeacon, I noticed that it was removed and I had to add another dependency in README.md.

Could you please suggest changes to the current build to make it compatible with bluez? Thanks for reporting this.

**Comment by thomwiggers** [comment]
> Could you please suggest changes to the current build to make it compatible with bluez? Thanks for reporting this.

I'm afraid not, I've got no clue how bluez works and I was just running PyBeacon to blast Rick Astley through the office :smile:. 

I just thought it would be a good idea to mark this as an (potential) issue.

**Comment by PrabhanshuAttri** [comment]
Thanks for reporting. Let me know if you encounter this issue while using PyBeacon. I sure someone will definitely find a solution to make PyBeacon compatible.

>  ... I was just running PyBeacon to blast Rick Astley through the office smile.

Haha, I used to bug my colleagues with my favorite restaurants on Zomato. :laughing:
